test/unit/uk/gov/hmrc/individualsbenefitsandcreditsapi/domain/integrationframework/IfAwardSpec.scala

```diff
@@ -25,6 +25,7 @@ import unit.uk.gov.hmrc.individualsbenefitsandcreditsapi.utils.UnitSpec
 class IfAwardSpec extends UnitSpec {
 
   val awards: IfAward = IfAward(Some("2020-08-18"),
+                       Some("2020-08-18"),
                        Some("2020-08-18"),
```

Comment:
It is a bit of a shame these now three fields use the same date, leaving the possibility of a positional error.  WTCAwardSpec and the other use 3 distinct dates

Reply:
postedDates updated, I'm not touching other dates for now...

Reply:
But postDated is set to Some("2020-08-18"), the same as the other positional parameters, it would be better if it had another value

Reply:
Ah sorry, I did not scroll down enough, yes I am now fine with the change from "2020-08-18" to "2020-08-19" for postedDate there